### ISSUE TITLE
Avoid downloading the whole dataset when only README.me has be touched on hub.

### ISSUE THREAD

**Issue by zinc75** [huggingface/datasets#6929]
### Feature request

`datasets.load_dataset()` triggers a new download of the whole dataset when the README.md file has been touched on huggingface hub, even if data files / parquet files are the exact same.

Is there a clever way to only download again the dataset **if** data is modified ?  

I understand the load_dataset function is triggered with a change of the hash of commits on huggingface, 

### Motivation

The current behaviour is a waste of network bandwidth / disk space / research time.

### Your contribution

I don't have time to submit a PR, but I hope a simple solution will emerge from this issue !

**Comment by severo** [comment]
you're right, we're tackling this here: https://github.com/huggingface/dataset-viewer/issues/2757

**Comment by zinc75** [comment]
@severo : great !
